modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -799,7 +829,8 @@ CV_EXPORTS bool detectQRCode(InputArray in, std::vector<Point> &points, double e
     @param decoded_info String information that is encrypted in QR code.
     @param straight_qrcode Matrix of the type CV_8UC1 containing an binary straight QR code.
     */
-CV_EXPORTS bool decodeQRCode(InputArray in, InputArray points, std::string &decoded_info, OutputArray straight_qrcode = noArray());
+CV_EXPORTS bool decodeQRCode(InputArray in, InputArray points, std::string &decoded_info, OutputArray straight_qrcode);
```

Comment:
`straight_qrcode` has default parameter `noArray` which means that this parameter could be not necessary